internal/cli/cli.go

```diff
@@ -6,9 +6,12 @@ import (
 	"fmt"
 	"log"
 	"os"
+
+	pus "github.com/innosat-mats/rac-extract-payload/internal/innosatPUS"
```

Comment:
why different name between directory and package, and by having them in internal it signals that it's not suitable for others to use in their projects and isn't rather the opposite, that the "pus" part could be used with anyone that has a mats rac-data file at hand? 

I also don't like the name PUS or innosatPUS since innosat is just a company right? and PUS is just Packet Utilization Standard. Neither of which describes the contents. I think something like "payload" would be much more sensible.

Reply:
Also how does this work before the code is at github?
And when you are working on a newer version of the package, locally, which version is found if the cli is run/build?
On your branch in VSC, I can't inspect definitions of the things in the pus package, neither renaming works. Have you done anything to the configuration, and is it possible to check in those setting?